.github/workflows/validate-python.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - name: 'Validate tests'
         working-directory: ./cbioportal-core
         run: |
-          docker run -v ${PWD}:/cbioportal-core python:3.6 /bin/bash -c '
+          docker run -v ${PWD}:/cbioportal-core python:3.6 /bin/sh -c '
             cd cbioportal-core &&
             pip install -r requirements.txt &&
-            source test_scripts.sh'
+            ./test_scripts.sh'
```

Comment:
This may not matter, but running test_scripts.sh like this invokes it in a subshell rather than in the current (shell) process.
If there were environment variables set by the script which need to be exported or read out they would be lost this way.

Reply:
indeed. If we lose any relevant env. variable, tests will fail.
`source` command is bash specific btw